src/components/Node/Node.js

```diff
@@ -371,6 +371,7 @@ export default memo(function Node({
 
 const ThemedNode = styled.div`
   display: ${({ theme }) => theme.nodeDisplay};
```

Comment:
can we call everything node or card?

Reply:
so should we use "Node" then?, because they files are called that

Reply:
yah fine